be/src/common/config.h

```diff
@@ -852,6 +852,10 @@ CONF_Int64(object_storage_request_timeout_ms, "-1");
 CONF_Strings(fallback_to_hadoop_fs_list, "");
 CONF_Strings(s3_compatible_fs_list, "s3n://, s3a://, s3://, oss://, cos://, cosn://, obs://, ks3://, tos://");
 CONF_mBool(s3_use_list_objects_v1, "false");
+CONF_String(jindo_fs_test_endpoint, "");
+// for jindo test only
+CONF_Strings(jindo_fs_list, "");
```

Comment:
`jindo_fs_list` looks like is not test only
Users have to configure `oss://` here so they can use jindo

Reply:
Yep.
 
`jindo_fs_list` is used to activate jindo, like the corresponding `s3_compatible_fs_list` one. 
Jindo can read ordinary oss in their own, or to avoid unindented behavior, user can provide a more specified intercept list to use jindo(like `oss://some.bucket.at.endpoing` or event `oss://some.bucket.at.endpoing/some/file` )

Reply:
Oh I see, I misplace the comment.

Reply:
Should be corrected.